### DISCUSSION TITLE
Prisma engines not downloading

### DISCUSSION THREAD

**Discussion by benschool** [prisma/prisma#20020] [Q&A]
After doing an `npm update` to update all my packages, my prisma upgraded to 4.16.1 so I needed to regenerate the client. 
Running `npx prisma generate --schema src/prisma/schema.prisma` results in the following terminal output:
```
> prisma generate --schema src/prisma/schema.prisma
> Downloading Prisma engines for Node-API for darwin [==========          ] 52%%
```
It never seems to be able to download it, gets halfway then just quits with no error and no generated client. Can I manually download them or something? I am not in an office and have tried several networks. I am able to download other things so I'm not sure why there would be something blocking the download, especially at 50%.

**Comment by nurul3101** [comment]
Hello @benschool 👋 

The latest prisma version is `4.16.2`. Can you try updating prisma to version `4.16.2` and check if the process still gets stuck?

**Reply by nurul3101** [reply]
@benschool I am checking to see if you had a chance to look at my last response.

It would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 
Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏

**Reply by nurul3101** [reply]
We are closing this Discussion due to inactivity. Please open a new Discussion if you have a similar problem 🙏
